shared/actions/chat2/index.tsx

```diff
@@ -2807,6 +2808,26 @@ const onChatMaybeMentionUpdate = (state, action: EngineGen.Chat1ChatUiChatMaybeM
   })
 }
 
+const onChatGetCoordinate = (state, action: EngineGen.Chat1ChatUiChatGetCoordinatePayload) => {
```

Comment:
this should go into actions/platform-specific

Reply:
Not sure we need to do that, the API is the same on both platforms, they made a conscious decision to do that on the RN side it seems.